app/design/frontend/Magento/luma/Magento_Catalog/web/css/source/_module.less

```diff
@@ -998,13 +998,23 @@
                 }
             }
         }
+
+        .block-compare {
+            .action {
+                &.delete {
+                    left: 0;
+                    right: initial;
+                }
+            }
+        }
     }
```

Comment:
Can you check delete action in smaller view in luma theme ? Current seem everything good with me

Reply:
Haven't found any issue.

Reply:
seem everything work ok in luma now :)